### ISSUE TITLE
Textarea for rule script editor has wrong height

### ISSUE THREAD

**Issue by sja** [eclipse/smarthome#2312]
I already have a solution but no time to fix that line:

https://github.com/eclipse/smarthome/blob/9f862e59309bc8424eec5fc1fb5c97e030a62a23/extensions/ui/org.eclipse.smarthome.ui.paper/web-src/css/components.css#L694

There is `overflow:scroll` but to fix the height calculation for an auto-growing textarea this has to be `overflow:hidden`.

I think the solution here was copied from [this codepen](https://codepen.io/vsync/pen/czgrf) and there it is also that case.

**Comment by aounhaider1** [comment]
No, actually that's the desired behaviour. The textarea should only expand to a maximum height and after that, the scroll should be available and height should not increase.

**Comment by sja** [comment]
Then you have to implement that. Actually, in Chrome this is unusable!

**Comment by aounhaider1** [comment]
Is it not working at all for you? Can you please share a screenshot and the chrome version that you are using? I am currently testing it with 53.* and 54.*.

**Comment by kaikreuzer** [comment]
@sja?

**Comment by sja** [comment]
It's still the same. In the mean time it's Chrome 55 in macOS 10.12. System setting: Always show scroll bars. The following screenshot shows a two-line script:
https://www.dropbox.com/s/b0uoteghyx4l41r/Screenshot%202017-01-24%2008.00.56.png?dl=0
